internal/discover/graphics.go

```diff
@@ -271,13 +274,21 @@ func newXorgDiscoverer(logger logger.Interface, driver *root.Driver, nvidiaCTKPa
 		lookup.NewFileLocator(
 			lookup.WithLogger(logger),
 			lookup.WithRoot(driver.Root),
-			lookup.WithSearchPaths(libRoot, "/usr/lib/x86_64-linux-gnu"),
+			lookup.WithSearchPaths([]string{
+				libRoot + "/nvidia/xorg",
```

Comment:
If `libcuda.so.<RM_VERSION>` is at `/usr/lib/x86_64-linux-gnu`, for example, libRoot will be `/usr/lib/x86_64-linux-gnu` is this the intent? Or should this always be `/usr/lib/...` or `/usr/X11R/...`?

Reply:
Yes. This is the intent. I removed `"/usr/lib/x86_64-linux-gnu"` because it's not arch-neutral and redundant with libroot.

In Ubuntu's APT installation for NVIDIA drivers, the paths where these packages are installed resolve to `/usr/lib/x86_64-linux-gnu/nvidia/xorg` for both X11-related .so files.